lemur/plugins/lemur_acme/plugin.py

```diff
@@ -431,6 +452,19 @@ def get_dns_provider(self, type):
             raise UnknownProvider("No such DNS provider: {}".format(type))
         return provider
 
+    def get_cname(self, domain):
+        """
+        :param domain: Domain name to look up a CNAME for.
+        :param record_type: Type of DNS record to lookup.
```

Comment:
`record_type` can be removed if not needed.

Reply:
Thanks!